### PR TITLE
Feedback: deprecating RAP and Page details 2.0

### PR DESCRIPTION
**Pull request by Garneauma**
- Deprecating RAP
- Deprecating Page details 1.1 and 2.0
- Updating code samples with correct characters

### CODE REVIEW

File: sites/page-details/index.json-ld
Comment (duboisp) [inline sites/page-details/index.json-ld:1]: Deprecate the version 2.x and make the version 2.1 to be the version 3.0 of the component.

Remove the implementation of version 2 and create its own change set by duplicating the one you created.

File: sites/feedback/index.json-ld
Comment (duboisp) [inline sites/feedback/index.json-ld:1]: Through the implementation, this "version 2.1" need to be change for "version 3.0" in regard of the page details.

**Review by duboisp — CHANGES_REQUESTED** [review]
2 little change request. 

Other than that its all good. I reviewed all the code and tested locally.

**Review by duboisp — CHANGES_REQUESTED** [review]
One little change because we released page-details version 3.

**Comment by Garneauma** [comment]
> One little change because we released page-details version 3.

Done. Sorry, I missed that.

**Pull request merged**